### PR TITLE
Weno bug fixes and missing files update

### PR DESCRIPTION
**Pull request by juggernautsei**
Fixes #

#### Changes proposed in this pull request:
These bug fixes are centered around the log sync functionality.

### CODE REVIEW

**Comment by sjpadgett** [comment]
@juggernautsei From what I here Weno has cut back on signup pricing. Is this true?

**Comment by juggernautsei** [comment]
> @juggernautsei From what I hear Weno has cut back on signup pricing. Is this true?

Yes, they have changed. It is now $350 after the practice has completed testing and is ready to go live.

**Comment by sjpadgett** [comment]
That's terrific, good on you.

**Comment by WENO-eRx** [comment]
WENO has lowered their EZ Integration price to $300 but still requires the payment upon registration at online.wenoexchange.com - however it will soon not be required until Test Mode is requested to be turned off (About June 14, 2021).  

Meanwhile, WENO can do a workaround and register EZ integration accounts on the back end for no charge.  Just ask.

**Comment by juggernautsei** [comment]
Please make sure all of these make it into rel 6_0_0_1

image

File: interface/weno/logsync.php [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2021 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+require_once($GLOBALS["srcdir"] . "/formatting.inc.php");
```
Comment (bradymiller) [inline interface/weno/logsync.php:12]: formatting.inc.php is already automatically included whenever bring in globals.php, so can remove above line

File: interface/weno/logsync.php [hunk 2]
```diff
@@ -0,0 +1,119 @@
+<?php
+
```
Comment (bradymiller) [inline interface/weno/logsync.php:2]: where is this script (logsync.php) called in OpenEMR???

File: interface/weno/logsync.php [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+    $rpt = curl_exec($ch);
+    if (curl_errno($ch)) {
+        throw new Exception(curl_error($ch));
+    }
+    $statusCode = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+    curl_close($ch);
+    if ($statusCode == 200) {
+        file_put_contents($logsync->rxsynclog, $rpt);
+        $logstring = "prescrition log import initiated successfully";
```
Comment (bradymiller) [inline interface/weno/logsync.php:54]: mispell here, should be:
```
prescription log import initiated successfully
```

File: library/weno_log_sync.php [hunk 4]
```diff
@@ -8,6 +8,7 @@
  *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
+require_once "../interface/globals.php";
 use OpenEMR\Rx\Weno\Container;
```
Comment (bradymiller) [inline library/weno_log_sync.php:12]: place a newline between the `require_once` line and the `use` line

**Comment by bradymiller** [comment]
@juggernautsei ,
Noted couple issues above. I can address all of them (since expediting this) tomorrow except for the interface/weno/logsync.php issue since I can not see anywhere in OpenEMR where that script is called. Am I missing something with that?

File: interface/weno/logsync.php (hunk 3 above)
Comment (bradymiller) [reply interface/weno/logsync.php:54]: i also noted 2 other places in codebase in src/Rx/Weno/LogProperties.php where this same string is misspelled. Should that also be fixed?

Comment (bradymiller) [reply interface/weno/logsync.php:54]: Or should we keep these misspellings (ie. is there something dependent on this string upstream or downstream)?

**Comment by juggernautsei** [comment]
@bradymiller, can hold one more day. The fixes did not fix the log sync. Still can't figure out why it not working. Fed and I will be working on it today.

**Comment by juggernautsei** [comment]
This is obsolete as of tonight.

**Pull request closed without merging**